recipes/libdicom/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "libdicom" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ImagingDataCommons/libdicom/releases/download/v{{ version }}/libdicom-{{ version }}.tar.xz
+  sha256: 3b88f267b58009005bc1182d8bd0c4a3218013ce202da722e5e8c9867c6f94f4
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
have you tried windows or just no interest from yourself?

Reply:
i haven't tried because i am not currently at a windows machine. libdicom should compile for windows though. would a `build.bat` script need to be added, or would `build.sh` be used?

Reply:
`bld.bat` would be used, since openslide build for windows today, we can leave this as future work.

Reply:
sounds great, let's incorporate windows builds in the future